packages/client/bin/cli.ts

```diff
@@ -292,6 +295,10 @@ const args: ClientOpts = yargs(hideBin(process.argv))
       'To run client in single node configuration without need to discover the sync height from peer. Particularly useful in test configurations. This flag is automically activated in the "dev" mode',
     boolean: true,
   })
+  .option('loadBlocksFromRlp', {
+    describe: 'path to a file of RLP encoded blocks',
```

Comment:
We should take a look how other clients import these blocks. I would be more in favor to only allow this if some kind of "test network" flag is provided.

Reply:
It's pretty similar. They look for a specific flag and load them if so. Why would this be an issue for us as currently designed? This flag never gets set for a production run.